src/js/communities/community-data.js

```diff
@@ -0,0 +1,95 @@
+export const CommunityData = [{
+    name: 'Newcastle Coders Group',
+    location: "ICT Building, University of Newcastle",
+    period: {
+        interval: "Monthly",
+        //maybe use a single digit here and convert it later (i.e. "2" becomes "2nd")
+        day: "2nd",
+        weekDay: "Wed"
+    },
+    //some of these groups only have a handful of links
```

Comment:
Could you please move these comments to where the links are actually generated maybe line 17 in the community card file [here](https://github.com/newwwie/newwwie.com/pull/81/files#diff-9a2b79eb717319911e2ecb2ab242be2d0008c77525496d01b604bc3ae11b2958R17) just so it makes sense in context.

Reply:
Oh sorry, I thought I removed those.

Reply:
Yeah that's fair. I'll fix the formatting as well.